scripts/ci/cmake/ci-ubuntu20.04-gcc8-static-serial.cmake

```diff
@@ -8,6 +8,7 @@ BUILD_TESTING:BOOL=ON
 ADIOS2_BUILD_EXAMPLES:BOOL=OFF
 
 ADIOS2_USE_AWSSDK:STRING=OFF
+ADIOS2_USE_BigWhoop:STRING=OFF
```

Comment:
Why are we failing the static builds with this?

Reply:
Not sure, if I understand your question. Do you mean the failing 'sanitizer' test? That one is timing out. Maybe a forced restart can rerun the test and see if it still occurs.

Reply:
This file is not consumed by the sanitizer jobs, this is consume by the static builds jobs. We need to make sure that this change does not break out static builds

Reply:
As suggested by @eisenhauer, we should leave out the mods on this file, i.e., rolling back that line, until it really needs to change.

Reply:
I am still opposed to this change, this variable is `AUTO` meaning that it will not enable bigwhoop if is not installed in that particular job image.

Reply:
Hi @vicentebolea ,
I am still not sure whether I can follow your thoughts on this. Regarding the initial questions, I could not find the static builds to be failing. 
I removed the line, but I might have been to quick on that manner regarding your last comment. Modifications on this originally were meant to conform with how other thirdparty libraries and, in particular, compressors are turned off in that file. 
I find it reasonable to introduce that change only if it is really necessary, which is why I removed the line again. But I am open for either option.

Reply:
@gregorweiss this file controls a static build job. 

> Modifications on this originally were meant to conform with how other thirdparty libraries and, in particular, compressors are turned off in that file.

Fair point, we disable compressors there to keep the build lean (static builds can exceed github actions limits). Still if it is not installed there is no need to disable it.